packages/contracts-bedrock/snapshots/storageLayout/L1StandardBridge.json

```diff
@@ -36,24 +22,45 @@
   },
   {
     "bytes": "20",
-    "label": "messenger",
+    "label": "spacer_3_0_20",
     "offset": 0,
     "slot": "3",
-    "type": "contract ICrossDomainMessenger"
+    "type": "address"
   },
   {
     "bytes": "20",
-    "label": "otherBridge",
+    "label": "spacer_4_0_20",
     "offset": 0,
     "slot": "4",
-    "type": "contract StandardBridge"
+    "type": "address"
   },
   {
-    "bytes": "1440",
+    "bytes": "1408",
     "label": "__gap",
     "offset": 0,
     "slot": "5",
-    "type": "uint256[45]"
+    "type": "uint256[44]"
+  },
+  {
+    "bytes": "1",
+    "label": "_initialized",
+    "offset": 0,
+    "slot": "49",
+    "type": "uint8"
+  },
+  {
+    "bytes": "1",
+    "label": "_initializing",
+    "offset": 1,
+    "slot": "49",
+    "type": "bool"
+  },
+  {
+    "bytes": "30",
+    "label": "spacer_49_2_30",
+    "offset": 2,
+    "slot": "49",
+    "type": "bytes30"
```

Comment:
The gap is reduced by one, which makes space for `_initialized/_initializing` to consume a full slot (along with a spacer).
Subsequent layout is unaffected by this change.